cookbook/deployment/cluster/config_flyte_deploy.rst

```diff
@@ -0,0 +1,181 @@
+
+.. tip::
+    The equivalent command in ``flytectl`` is:
+
+    .. prompt:: bash
+        flytectl update execution-queue-attribute
+
+    Refer to the :ref:`docs <flytectl:flytectl_update_execution-queue-attribute>` to learn more about the command and its supported flag(s).
+
+You can view existing attributes for which tags can be assigned by visiting ``protocol://<host>/api/v1/matchable_attributes?resource_type=3``.
+
+Execution Cluster Label
+=======================
+This allows forcing a matching execution to consistently execute on a specific Kubernetes cluster.
+
+You can set this using flyte-cli:
+
+.. prompt:: bash
+
+   flyte-cli -h localhost:30081 -p flyteexamples -d development update-execution-cluster-label --value mycluster
+
+.. tip::
+   The equivalent command in ``flytectl`` is:
+
+   .. prompt:: bash
+      flytectl update execution-cluster-label
+
+   Refer to the :ref:`docs <flytectl:flytectl_update_execution-cluster-label>` to learn more about the command and its supported flag(s).
+
+*********
+Hierarchy
+*********
+Increasing specificity defines how matchable resource attributes get applied. The available configurations, in order of decreasing specifity, are:
+
+#. Domain, project, workflow name, and launch plan.
+
+#. Domain, project, and workflow name
+
+#. Domain and project
+
+#. Domain
+
+Default values for all and per-domain attributes may be specified in the FlyteAdmin config as documented in the :ref:`config_resource_types`.
+
+Example
+=======
+If the database includes the following:
+
++------------+--------------+----------+-------------+-----------+
+| Domain     | Project      | Workflow | Launch Plan | Tags      |
++============+==============+==========+=============+===========+
+| production | widgetmodels |          |             | critical  |
++------------+--------------+----------+-------------+-----------+
+| production | widgetmodels | Demand   |             | supply    |
++------------+--------------+----------+-------------+-----------+
+
+Any inbound ``CreateExecution`` requests with **[Domain: Production, Project: widgetmodels, Workflow: Demand]** for any launch plan will have a tag value of "supply".
+
+Any inbound ``CreateExecution`` requests with **[Domain: Production, Project: widgetmodels]** for any workflow other than ``Demand`` and any launch plan will have a tag value "critical".
+
+All other inbound CreateExecution requests will use the default values specified in the FlyteAdmin config (if any).
+
+*********
+Debugging
+*********
+Use the `get <https://github.com/flyteorg/flyteidl/blob/ba13965bcfbf7e7bfce40664800aaf1f2a1088a1/protos/flyteidl/service/admin.proto#L395>`__ endpoint
+to see if overrides exist for a specific resource. Hit the URL: ``protocol://<host/api/v1/project_domain_attributes/widgetmodels/production?resource_type=2>``.
+
+To get the global state of the world, list all endpoints. For example, visit ``protocol://<host>/api/v1/matchable_attributes?resource_type=2``.
```

Comment:
@pmahindrakar-oss more potential flytectl commands here?

Reply:
Same here . resource_type=2 is i 
```
// Configures task and dynamic task execution queue assignment.
MatchableResource_EXECUTION_QUEUE MatchableResource = 2
```
And hence can be referenced using this https://docs.flyte.org/projects/flytectl/en/latest/gen/flytectl_update_execution-queue-attribute.html